build/configs/imxrt1020-evk/imxrt1020-evk_download.sh

```diff
@@ -18,64 +18,21 @@
 ###########################################################################
 # imxrt1020-evk_download.sh
```

Comment:
Could you shorten the commit title please?

> build/configs/imxrt1020-evk : Add common download script for _imxrt102…
> …0-evk_ 

can remove the board name since the path is descriptive already.

Reply:
Done.